cicd/main.yml

```diff
@@ -0,0 +1,110 @@
+#trigger:
+#  batch: true
+#  branches:
+#    include:
+#      - refs/heads/master
+#  paths:
+#    exclude:
+#    - README.md
+
+pool: Computer-Vision-Team-Amsterdam
+
+parameters:
+  - name: AMLModelVersion
+    displayName: Version of the OOR model in AML
+    type: string
+    default: 1
+
+resources:
+  repositories:
+    - repository: self
+
+variables:
+  - template: variables.yaml
+
+stages:
+  - stage: Prebuild
+    displayName: "Collect all prerequisite and run tests prior to build"
+    jobs:
+      - template: "prebuild-pipeline.yml"
+        parameters:
+          serviceConnectionName: ${{ variables.serviceConnectionName }}
+          AMLModelName: ${{ variables.AMLModelName }}
+          AMLModelVersion: ${{ parameters.AMLModelVersion }}
+          AMLWorkspaceName: ${{ variables.AMLWorkspaceName }}
+          AMLResourceGroup: ${{ variables.AMLResourceGroup }}
+          AMLSubscriptionID: ${{ variables.AMLSubscriptionID }}
+          downloadedModelDirectory: ${{ variables.downloadedModelDirectory }}
+          publishedModelArtifactsName: ${{ variables.publishedModelArtifactsName }}
+
+  - stage: Build_ont
+    displayName: "[ont] Build and publish container to ont"
+    dependsOn:
+      - Prebuild
+    condition: |
+      and
+      (
+        succeeded(),
+        eq( variables.dtapName, 'ont' ),
+        ne( variables['Build.Reason'], 'PullRequest' )
+      )
+    variables:
+      - name: projectVersion
+        value: $[ stageDependencies.Prerequisites.GetProjectVersion.outputs['getProjectVersion.PROJECT_VERSION'] ]
```

Comment:
this should be  value:
` $[stageDependencies.Prebuild.GetProjectVersion.outputs['getProjectVersion.PROJECT_VERSION'] ]`

Reply:
Nice spot!